src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -461,6 +462,14 @@ public String latestVersionId() {
         return latest != null ? latest.id : null;
     }
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonView(JsonViews.UserInterface.class)
+    @JsonProperty("latestVersionEndDate")
+    public LocalDate latestVersionEndDate() {
+        FeedVersion latest = retrieveLatest();
```

Comment:
Does `retrieveLatest` retrieve the latest **deployed** version? If so, would it be possible to also expose the id of this version?

Reply:
If I have understood correctly, please see latest update.